### DISCUSSION TITLE
environment config as commandline parameter

### DISCUSSION THREAD

**Discussion by christian-inzko-dt** [Dynatrace/dynatrace-configuration-as-code#371] [Ideas]
I am trying to use the monaco tool for loading some initial data (e.g. dashboards. MZs, ...) into newly created loadtest clusters. But from the documentation and tests i have done, the tool always requires an enviornment yaml containing the environment url and token name which makes it more complicate to be used for my usecase (i always have to create an environment file containing the url and name for the newly created environment
Would it be possible to skip this environment file and provide the infos (url and token name) just via commandline parameters?

**Comment by patrickpichler** [comment]
This is currently not possible and it will currently not be possible
with monaco v2. 

In monaco v2 the config model will be completely reworked and 
the `environments.yaml` will be promoted to a deployment manifesto.
So it takes an even more vital role in monaco. 

For more info see the ticket #351  and the corresponding discussion in 
#165 
The new config model is pretty much done. All work is done in PR #261.
Documentation on how this new config model work can be found [here](https://github.com/patrickpichler/dynatrace-monitoring-as-code/blob/poc-new-config/documentation/docs/configuration/configuration.md).
